js/languagePickerView.js

```diff
@@ -22,7 +22,8 @@ define([
             var data = this.model.toJSON();
             var template = Handlebars.templates[this.constructor.template];
             this.$el.html(template(data));
```

Comment:
yes noticed this after I saved the comment and updated the comment to update the `className` attribute.